src/Wellcome.Dds/Wellcome.Dds.AssetDomainRepositories.Tests/DlcsSynchronisation/ProcessingBehaviourTests.cs

```diff
@@ -147,23 +154,23 @@ public void Access_MP4_1440_Has_iiif_av_and_file_Delivery_Channel()
         var pf = MakePhysicalFile();
         pf.Files = new List<IStoredFile>
         {
-            new StoredFile { MimeType = "video/mp4" },       // the access copy
-            new StoredFile { MimeType = "application/mxf" }  // the master
+            new StoredFile { MimeType = "video/mp4", PhysicalFile = pf},       // the access copy
+            new StoredFile { MimeType = "application/mxf", PhysicalFile = pf }  // the master
         };
         pf.MimeType = "video/mp4";
-        SetHeight(pf, 1440);
+        SetHeight(pf.Files[0], 1440);
 
-        var processing = pf.ProcessingBehaviour;
+        var processing = pf.Files[0].ProcessingBehaviour;
 
         processing.DeliveryChannels.Should().OnlyContain(s => s == "iiif-av" || s == "file");
```

Comment:
Similar to above with "thumbs", this will pass if it's only "iiif-av" or "file".

```cs
processing.DeliveryChannels.Should().HaveCount(2).And.OnlyContain(s => s == "iiif-av" || s == "file");

// OR 
var expected = new HashSet<string> { "iiif-av", "file" };
processing.DeliveryChannels.Should().BeEquivalentTo(expected);
```